ruby/tests/common_tests.rb

```diff
@@ -708,6 +708,28 @@ def test_enum_lookup
     assert proto_module::TestEnum::resolve(:C) == 3
   end
 
+  def test_repeated_push
+    m = proto_module::TestMessage.new
+
+    m.repeated_string << "one"
+    m.repeated_string.push "two"
+    assert_equal ["one", "two"], m.repeated_string
+
+    m.repeated_string.push ["three", "four"]
```

Comment:
This should only behave this way if instead of an array, there values were given as different arguments to `push`:

```ruby
m.repeated_string.push "three", "four"
# or
m.repeated_string.push *["three", "four"]
```

Reply:
I reverted the change to the other operations and changed `push` to allow multiple arguments. Let me know if that looks right.